README.md

```diff
@@ -42,12 +42,12 @@ architecture:
    updated by returning a new value.
 3. Stores do not contain data, they _transform_ it. See the section on
    stores below.
-4. All Actions that return promises will wait to resolve before
-   dispatching.
+4. Actions must either return a value, promise, or follow the
+   error-first callbacks convention, dispatching on success.
```

Comment:
a code example of "or follow the  error-first callbacks convention, dispatching on success." might be nice.